e2e_tests/integration/viz.spec.ts

```diff
@@ -20,6 +20,10 @@
 
 /* global Cypress, cy, before */
 
+const greyColor = 'rgb(165, 171, 182)' // Default color for nodes and relationships
```

Comment:
Should constants be uppercase?
```suggestion
const GREY_COLOR = 'rgb(165, 171, 182)' // Default color for nodes and relationships
```